integrations/hotjar/test/index.unit.test.js

```diff
@@ -124,5 +115,32 @@ describe('Hotjar Unit', function() {
         analytics.didNotCall(window.hj);
       });
     });
+
+    describe('#track', function() {
+      beforeEach(function() {
+        analytics.stub(hotjar, 'debug');
+        analytics.stub(window, 'hj');
+      });
+
+      afterEach(function() {
+        analytics.reset();
+      });
+
+      it('should send event and ignore properties', function() {
+        analytics.stub(window, 'hj');
+        var event = 'the_event';
+        var properties = { a: 'a', b: 'b', c: [] };
+        analytics.track(event, properties);
+        analytics.called(window.hj, 'event', event);

```

Comment:
Will that test fail if we suddenly started sending properties (ideally it should imo)?

Reply:
I've added a `didNotCall` to assert that we're not including the properties for the time being.